Source/ArmTemplate.json

```diff
@@ -42,6 +41,30 @@
       }
     },
 
+    // Create an identity to run the AzureSqlBulkFailover script. 
+    {
+      "type": "Microsoft.ManagedIdentity/userAssignedIdentities",
+      "apiVersion": "2018-11-30",
+      "name": "AzureSqlBulkFailoverRunbookIdentity",
+      "location": "[resourceGroup().location]"
+    },
+
+    // Grant the runbook identity Contributor role in this subscription. 
+    {
+      "type": "Microsoft.Authorization/roleAssignments",
```

Comment:
There's no 'scope' property. Does this mean that the default scope is the subscription? Just wanted to understand the behavior and confirm that it was your intent.